packages/css/src/components/header/header.scss

```diff
@@ -83,7 +94,6 @@
 
 @mixin header-menu-action {
   color: var(--ams-header-menu-item-color);
-  font-family: var(--ams-header-menu-item-font-family);
```

Comment:
The menu button no longer uses our font; buttons inherit styles selectively.

Reply:
Sharp! Do we want to add `font-family: inherit` to `.ams-header__mega-menu-button`? Or do we want to allow users to specify a different font for the button only?

Reply:
A different font for the button would be crazy, let’s go with inheritance :)